Added doc for eventing-sink test function (#16946)
kyma-project/kyma: tests/fast-integration/eventing-test/assets/README.md

# Eventing Sink

## Overview

> This eventing sink function is used to test the end-to-end flow for Eventing. It is responsible for three tasks:

Reviewer: ```suggestion
Use the `eventing-sink` Function to test the end-to-end flow for Eventing. 
It performs three tasks:
```

If this is about a Serverless Function, we should [capitalize](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#camel-case) it.

> 1. Forward requests to publish events to event-publisher-proxy.

Reviewer: If the order of list elements is important, keep the numbered list. 
If the order does not matter, use a bullet list.

Author: Updated it to use bullet list.

2. Save all received events in memory.
3. Return saved events to users.

## Usage

> ### Forward requests to publish events to event-publisher-proxy.

Reviewer: ```suggestion
### Forward requests to publish events to `event-publisher-proxy`
```

Headlines don't need a full stop.

> To publish an event to event-publisher-proxy, send a Http request with params `send=true` and body:

Reviewer: ```suggestion
To publish an event to `event-publisher-proxy`, send an HTTP request with parameter `send=true` and the following body:
```

```
{
    url: `http://eventing-event-publisher-proxy.kyma-system/publish`,
    data: {
        headers: {}, // Event headers
        payload: {}  // Event payload
    },
}
```

> ### Fetch an event received by eventing-sink from a backend.

Reviewer: ```suggestion
### Fetch an event received by `eventing-sink` from a backend
```

Headlines don't need a full stop

> To fetch an event received by eventing-sink, send a Http request with params `eventid=<id>`. The response contains:

Reviewer: ```suggestion
To fetch an event received by `eventing-sink`, send an HTTP request with parameter `eventid=<id>`. The response contains:
```
